### ISSUE TITLE
WhatsApp sharing does not work

### ISSUE THREAD

**Issue by gmajoulet** [ampproject/amphtml#13740]
Some publishers use a WhatsApp sharing option, that does not seem to work in some cases (Android? When the app isn't installed?).

See: www.cnn.com/ampstories/health/yes-sitting-too-long-can-kill-you-even-if-you-exercise

Sample code: 
```
<amp-social-share width="48" height="66" class="i-amphtml-story-share-icon i-amphtml-element i-amphtml-layout-fixed i-amphtml-layout-size-defined amp-social-share-whatsapp i-amphtml-layout" type="whatsapp" data-param-text="CNN: Yes, sitting too long can kill you, even if you exercise - https://www.cnn.com/ampstories/health/yes-sitting-too-long-can-kill-you-even-if-you-exercise" style="width: 48px; height: 66px;" role="button" tabindex="0">WhatsApp</amp-social-share>
```
Generates: 
```
whatsapp://send?text=CNN%3A%20Yes%2C%20sitting%20too%20long%20can%20kill%20you%2C%20even%20if%20you%20exercise%20-%20https%3A%2F%2Fwww.cnn.com%2Fampstories%2Fhealth%2Fyes-sitting-too-long-can-kill-you-even-if-you-exercise
```

Repros on: MacOS Chrome, Android Chrome (Pixel 2), feel free to add more.

**Comment by aghassemi** [comment]
@gmajoulet Since we like every issue to have an owner, assigning to you for now until you all decide who will work on it.

**Comment by matthias-erl** [comment]
I was even wondering wether the implementation of WhatsApp sharing via the `whatsapp://` protocol is still the way to go as `https://api.whatsapp.com/send?text=I'm%20inquiring%20about%20the%20apartment%20listing` seems to be the official according to [the docs](https://faq.whatsapp.com/en/android/26000030/?category=5245251).

**Comment by gmajoulet** [comment]
Thanks!

Closed by #13942
